### ISSUE TITLE
OSX Sierra: Super long latency when pressing first hotkey

### ISSUE THREAD

**Issue by clacke** [jigish/slate#482]
Since I upgraded to OSX 10.12 Sierra, it takes 9 seconds from pressing the hotkey to the window resizing, in an application that I just command-tabbed to. After the first resize, everything is normal. If I switch to another application, then switch back, the first resize takes 9 seconds again.

**Comment by clacke** [comment]
This is an intricate one. The problem went away when I switched from `color-theme-solarized` to `solarized-theme` in Emacs. I had the problem with every application (I suppose as long as Emacs was running), but changing this in Emacs, fixed the problem everywhere.

So this might not be a Slate problem as such. Still, I would be very interested if someone had a hunch why Slate was affected by an Emacs setting. Did it create a lot of windowing data that Slate had to slog through? Did it add any hooks? Emacs doesn't have accessibility access.

I'm also curious why I did not have this problem in OSX 10.11. I did not change my Emacs settings for a long time.
